### PR TITLE
Create terraform.yml

### PR DESCRIPTION
**Pull request by petrolifero**
#4

### CODE REVIEW

File: .github/workflows/terraform.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      run:
+        shell: bash
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: install Terraform
+      run: wget https://releases.hashicorp.com/terraform/0.15.5/terraform_0.15.5_linux_amd64.zip && unzip terraform_0.15.5_linux_amd64.zip && export PATH=$PATH:$(pwd)
+
```
Comment (dbofmmbt) [inline .github/workflows/terraform.yml]: Wouldn't it be better to use the Hashicorp's github action to install it?

https://github.com/marketplace/actions/hashicorp-setup-terraform

File: .github/workflows/terraform.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - name: Terraform Format
+      run: terraform fmt -check
+
+    - name: Terraform Plan
+      run: terraform plan
+
+    - name: Terraform Apply
+      if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+      run: terraform apply -auto-approve
```
Comment (dbofmmbt) [inline .github/workflows/terraform.yml]: I see that there's no human revision of the plan right on the PR in the worflow as it is. You can find this step in these references:

- https://learn.hashicorp.com/tutorials/terraform/github-actions?in=terraform/automation#review-actions-workflow
- https://github.com/marketplace/actions/hashicorp-setup-terraform

Basically, they take the output of the `terraform plan` command and show it directly on the PR to be easier to review. Do you think it is worth adding it?

Comment (petrolifero) [reply .github/workflows/terraform.yml]: This is great, I will add

File: .github/workflows/terraform.yml (hunk 1 above)
Comment (petrolifero) [reply .github/workflows/terraform.yml]: I think that was only to terraform cloud, but is not. so is implemented

File: .github/workflows/terraform.yml (hunk 2 above)
Comment (petrolifero) [reply .github/workflows/terraform.yml]: the github actions is done, i need to create a token on github. I will do this on monday.

**Review by dbofmmbt — DISMISSED** [review]

**Comment by dbofmmbt** [comment]
> #### Terraform Format and Style ``
> #### Terraform Initialization ``
> #### Terraform Plan ``
> Show Plan
> *Pusher: @petrolifero, Action: `pull_request`*

@petrolifero this comment looks a bit broken. Is it already fixed?

**Comment by petrolifero** [comment]
> 
> 
> > #### Terraform Format and Style ``
> > #### Terraform Initialization ``
> > #### Terraform Plan ``
> > Show Plan
> > _Pusher: @petrolifero, Action: `pull_request`_
> 
> @petrolifero this comment looks a bit broken. Is it already fixed?

Is broken. I will see this today (now, in fact)

**Comment by dbofmmbt** [comment]
Whitespace is important to markdown, indeed :laughing:

**Review by dbofmmbt — APPROVED** [review]

**Pull request merged**